ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -406,7 +406,8 @@ def record_process_sample(project, sample, workflow_subtask, analysis_module_nam
                                                                        e.message))
         extra_args = None
         if workflow_subtask == "merge_process_variantcall":
-            status_field = "alignment_status"
+            seqrun_status_field = "alignment_status"
+            sample_status_field = "UNDER_ANALYSIS"
```

Comment:
Ha dammit. Why do I code these so late at night.

I mean, I'm testing the PR reviewers.